File: 119. Pascal's Triangle II/119. Pascal's Triangle II.dart
class Solution {
  List<int> getRow(int rowIndex) {
    List<int> row = List<int>.filled(rowIndex + 1, 1);
    
    for (int i = 1; i <= rowIndex; i++) {
      int prev = row[0];
      for (int j = 1; j < i; j++) {
        int temp = row[j];
        row[j] = ((prev.toDouble() * (rowIndex - j + 1)) ~/ j).toInt();
        prev = temp;
      }
      row[i] = 1;
    }
    
    return row;
  }
}

File: 119. Pascal's Triangle II/119. Pascal's Triangle II.go
func getRow(rowIndex int) []int {
    row := make([]int, rowIndex+1)
    row[0] = 1
    
    for i := 1; i <= rowIndex; i++ {
        prev := row[0]
        for j := 1; j < i; j++ {
            temp := row[j]
            row[j] = prev * (rowIndex - j + 1) / j
            prev = temp
        }
        row[i] = 1
    }
    
    return row
}